libraries/CurieBLE/examples/test/CommunitySketches/StandardFirmataBLE/BLEStream.h

```diff
@@ -87,7 +87,10 @@ BLEStream::BLEStream(unsigned char req, unsigned char rdy, unsigned char rst) :
   this->_flushed = 0;
   this->_flushInterval = BLESTREAM_TXBUFFER_FLUSH_INTERVAL;
   BLEStream::_instance = this;
+}
 
+void BLEStream::begin(...)
```

Comment:
@sgbihu @SidLeung  please see my comments in https://github.com/01org/corelibs-arduino101/issues/377#issuecomment-271951484.